src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -246,21 +246,19 @@ protected enum MetricKind
 
         private const string RegisterName = nameof(SdcaTrainerBase<TArgs, TTransformer, TModel>);
 
-        private static TArgs ArgsInit(string featureColumn, SchemaShape.Column labelColumn, Action<TArgs> advancedSettings = null)
+        private static TArgs ArgsInit(string featureColumn, SchemaShape.Column labelColumn)
         {
             var args = new TArgs();
 
-            // Apply the advanced args, if the user supplied any.
-            advancedSettings?.Invoke(args);
             args.FeatureColumn = featureColumn;
             args.LabelColumn = labelColumn.Name;
             return args;
         }
 
         internal SdcaTrainerBase(IHostEnvironment env, string featureColumn, SchemaShape.Column labelColumn,
```

Comment:
>  internal SdcaTrainerBase [](start = 6, length = 26)

delete

Reply:
#2100

---
In reply to: [248770895](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248770895) [](ancestors = 248770895)